plugins/pulp_rpm/plugins/importers/yum/parse/rpm.py

```diff
@@ -2,6 +2,7 @@
 import logging
 import os
 import rpm as rpm_module
+import deltarpm
```

Comment:
I think according to pep8 this import as well as rpm one should go to the second group and it is also good to put them into the alphabetical order:

    import logging
    import os
    
    import deltarpm
    import rpm as rpm_module